src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
```

Comment:
As mentioned before, all classes will need to be adjusted to changes from `index.html`.